### PR TITLE
ADD: prisma with Project model

### LINKED ISSUE #95: [FEATURE] Integrate Prisma MongoDB into NextJS
### Description

Check references down below:

- **How to setup Prisma+MongoDB**: https://itnext.io/using-prisma-orm-with-mongodb-in-next-js-e42b1f7543e6

- **Prisma Official Docs**: https://www.prisma.io/docs/concepts/database-connectors/mongodb

### PR DESCRIPTION
**Pull request by ykhokhaneshiya**
fixes #95 

Integrated prisma in the project

### CODE REVIEW

File: prisma/prisma.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { PrismaClient } from '@prisma/client';
+
+let prisma;
+
+if (process.env.NODE_ENV === 'production') {
+  prisma = new PrismaClient();
+} else {
+  if (!global.prisma) {
```
Comment (rohitdasu) [inline prisma/prisma.js:8]: where is global coming from? just curious

Comment (rohitdasu) [reply prisma/prisma.js:8]: okay cool. thanks for clarification

Comment (RakeshSangem) [reply prisma/prisma.js:8]: Thanks for adding date object

Comment (rohitdasu) [reply prisma/prisma.js:8]: yeah, it's very important. Good catch

**Review by rohitdasu — APPROVED** [review]
Works fine! LGTM 👍

**Pull request merged**